microsoft-edge/webview2/release-notes/index.md

```diff
@@ -74,7 +74,108 @@ See also:
 <!-- July/August 2024 Release SDK -->
+## 1.0.2651.64
 
+Release Date: August 7, 2024
+
+[NuGet package for WebView2 SDK 1.0.2651.64](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2651.64)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 127.0.2651.64 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+The following APIs have been promoted to Stable and are now included in this Release SDK.
+
+
+<!-- ---------- -->
+* Updated the WebMessageObjects API to allow injecting DOM objects into WebView2 content that's constructed via the app, and via the `CoreWebView2.PostWebMessage` API in the other direction.  Added a new web object type to represent a file system handle that can be posted to the web content to provide it with filesystem access.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2` Class:
+   * [CoreWebView2.PostWebMessageAsJsonWithAdditionalObjects Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.postwebmessageasjsonwithadditionalobjects?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+
+* `CoreWebView2Environment` Class:
+   * [CoreWebView2Environment.CreateWebFileSystemDirectoryHandle Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createwebfilesystemdirectoryhandle?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+   * [CoreWebView2Environment.CreateWebFileSystemFileHandle Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createwebfilesystemfilehandle?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+
+* `CoreWebView2FileSystemHandle` Class:
+   * [CoreWebView2FileSystemHandle.Kind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandle.kind?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+   * [CoreWebView2FileSystemHandle.Path Property](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandle.path?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+   * [CoreWebView2FileSystemHandle.Permission Property](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandle.permission?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+
+* [CoreWebView2FileSystemHandleKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandlekind?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+   * `File`
+   * `Directory`
+
+* [CoreWebView2FileSystemHandlePermission Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2filesystemhandlepermission?view=webview2-dotnet-1.0.2651.64&preserve-view=true)
+   * `ReadOnly`
+   * `ReadWrite`
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2` Class:
+   * [CoreWebView2.PostWebMessageAsJsonWithAdditionalObjects Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.2651.64&preserve-view=true#postwebmessageasjsonwithadditionalobjects)
```

Comment:
PostWebMessageAsJsonWithAdditionalObjects not found in open PR for WinRT Ref docs.
It's not listed in last week's full build report.

PR for WinRT Ref docs:
https://github.com/MicrosoftDocs/webview2-winrt-reference/pull/61 - "temp" rendered nearby property:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.2651.64&branch=pr-en-us-61#postwebmessageasjson 
@peiche-jessica @victorhuangwq

Reply:
here's the Win32 link (temp modified) that i confirmed:
   * [ICoreWebView2_23::PostWebMessageAsJsonWithAdditionalObjects](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_23?view=webview2-win32-preview&branch=pr-en-us-94#postwebmessageasjsonwithadditionalobjects)

https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_23?view=webview2-win32-preview&branch=pr-en-us-94#postwebmessageasjsonwithadditionalobjects

Reply:
confirmed that draft commit has "Property".  changed to "Method".  hard to tell in RT docs, until inspect page structure.   draft for .NET correctly has "Method".  Win32 convention omits to specify.  @victorhuangwq

Reply:
Outcome: 
![image](https://github.com/user-attachments/assets/d4c6b9ed-5994-4cca-858a-9f27f22cd699)
@victorhuangwq